README.md

```diff
@@ -4,7 +4,7 @@
 **Tags:** pantheon, cdn, cache  
 **Requires at least:** 4.7  
 **Tested up to:** 6.2  
-**Stable tag:** 1.2.4  
+**Stable tag:** 1.3.0-dev 
```

Comment:
Just learned this— two spaces at the end of line line for the line break to appear properly (preview md to see it now looks weird)
```suggestion
**Stable tag:** 1.3.0-dev  
```